bundle.Dockerfile

```diff
@@ -4,14 +4,14 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=cluster-logging-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=4.7
-LABEL operators.operatorframework.io.bundle.channel.default.v1=4.7
+LABEL operators.operatorframework.io.bundle.channels.v1=5.1.beta1
+LABEL operators.operatorframework.io.bundle.channel.default.v1=5.1.beta1
```

Comment:
channel in downstream builds for TP will be `tech-preview`
Since this file is generated using `make generate-bundle` it shows `5.1.beta1`, and downstream builds do not use these generated files as of now